### ISSUE TITLE
Update quantity for updateCartItems mutation does not work

### ISSUE THREAD

**Issue by rogyar** [magento/graphql-ce#927]
### Description
Once an item has been added to the shopping cart, `updateCartItems` will not update the item quantity.

### Preconditions (*)
1. Create a simple product with SKU "simple"

### Steps to reproduce (*)
1. Create empty cart

```graphql
mutation {
  createEmptyCart
}
```

2. Add a simple product to the cart

```graphql
mutation {
  addSimpleProductsToCart(input: {
    cart_id: "$maskedCartId"
    cart_items: [
      {
        data: {
          quantity: 3,
          sku: "simple"
        }
      }
    ]
  }) {
    cart {
      items {
        id
        quantity
        product {
          name
        }
      }
    }
  }
}
```

3. Change the quantity of the added item using `updateCartItems` mutation

```graphql
mutation {
  updateCartItems(input: {
    cart_id:"$maskedCartId"
    cart_items: [
      {
        cart_item_id: $itemId
        quantity: 5
      }
    ]
  }) {
    cart {
      items {
        id
        quantity
        product {
          name
        }
      }
    }
  }
}
```

4. Retrieve the cart contents via separate query

```graphql
{
  cart(cart_id:"$maskedCartId") {
    items {
      id
      quantity
      product {
        name
      }
    }
  }
}
```

### Expected result (*)
1. The item quantity has been changed to 5

### Actual result (*)
1. The item quantity is still the initial one (3)

**Comment by rogyar** [comment]
Might be related to #792 
Checking the reason at the moment

**Comment by lenaorobei** [comment]
Fixed in `2.3.3-develop` branch.
